lab-anthony/gulpfile.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+
+gulp.task('lint', function () {
+  return gulp.src(['*.js', 'lib/*.js', '!node_modules/*js'])
+          .pipe(eslint())
+          .pipe(eslint.format());
+});
+
+gulp.task('default',['lint']);
+
+gulp.task('watch', function () {
+  const watcher = gulp.watch(['*.js', 'test/*.js', 'lib/*.js', '!node_modules/*js'], ['lint']);
+  watcher.on('change', function(e){
+    console.log('The file '+e.path+' was '+e.type+', running tasks again:');
```

Comment:
A good convention to follow is either to use es6 string interpolation or to [space out operators from items.](https://github.com/airbnb/javascript#whitespace--infix-ops) A '+' sign directly in front of a variable can concatenate it, but it can also in certain coerce a string into a number.